docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -942,6 +969,27 @@ Renders a block if a statement is false; does not support operators for comparis
 - [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/unless.js)
 - [See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=unless)
 
+
+### {{option}}
+
+```handlebars
+{{option "a.b.c"}}
+```
+Returns the given value of prop from `this.options`.
+
+### {{get}}
+```handlebars
+{{get (concat “a.b” “.c.d”) someObject}}
+```
+Use property paths (`a.b.c`) to get a value or nested value from the context. Works as a regular helper or block helper. You can use return values from the `concat` helper as property paths.

```

Comment:
```suggestion
Use property paths (`a.b.c.d`) to get a value or nested value from the context. Works as a regular helper or block helper. You can use return values from the `concat` helper as property paths.

```

Reply:
{{#get}} helper performs a “browse” query that creates a block of data that represents a list of your posts, authors, tags, or tiers.